README.rdoc

```diff
@@ -73,6 +73,14 @@ gitflow will automatically:
 * push the code and tags to the remote "origin"
 * and run the normal deploy task for the production stage.
 
+=== CLEANING OLD TAGS:
+Whenever you want to clean the old tags generated by this gem you can simply do
+
+    bundle exec cap <stage_name> gitflow:cleanup_tags
+
+This will delete all tags that have this pattern  [staging|production]{1}-[0-9]{4}-[0-9]{2}-[0-9]{2}\-([0-9]*)
```

Comment:
Might be worth linking to the source where this is defined, just because it'd be easy for this to get out of date if the implementation changes.